os/img/transpile-workflow.svg

```diff
@@ -0,0 +1 @@
0,1-.28.73,1,1,0,0,1-1.38,0A1,1,0,0,1,103.19,277.73Z" fill="#54a3d9"/><path d="M70.62,293h-.8l-.08-1h0a4.21,4.21,0,0,1-1,.84,2.24,2.24,0,0,1-1.21.32,1.88,1.88,0,0,1-1.54-.59,2.72,2.72,0,0,1-.49-1.76v-3.62h1v3.49a2,2,0,0,0,.31,1.23,1.23,1.23,0,0,0,1,.4,1.67,1.67,0,0,0,.91-.26,4.35,4.35,0,0,0,.9-.82v-4h1Z" fill="#54a3d9"/><path d="M72.88,287.16h.82l.08,1h0a4.67,4.67,0,0,1,1-.82A2.26,2.26,0,0,1,76,287a1.86,1.86,0,0,1,1.53.59,2.73,2.73,0,0,1,.49,1.76V293h-1V289.5a2,2,0,0,0-.31-1.23,1.23,1.23,0,0,0-1-.4,1.7,1.7,0,0,0-.49.07,1.86,1.86,0,0,0-.44.2,3.25,3.25,0,0,0-.44.33q-.23.2-.48.46V293h-1Z" fill="#54a3d9"/><path d="M80,287.16h3.7V293h-1v-5H80Zm3.1-1.2a.81.81,0,0,1-.57-.2.72.72,0,0,1-.22-.55.73.73,0,0,1,.22-.56.87.87,0,0,1,1.13,0,.72.72,0,0,1,.23.56.71.71,0,0,1-.23.55A.82.82,0,0,1,83.14,286Z" fill="#54a3d9"/><path d="M88.64,288H87v-.74l1.68-.06.13-1.63h.82v1.63h2.87v.8H89.62v2.77a2.94,2.94,0,0,0,.07.69,1.23,1.23,0,0,0,.24.5,1,1,0,0,0,.46.31,2.12,2.12,0,0,0,.71.1,3.64,3.64,0,0,0,.76-.07,4.31,4.31,0,0,0,.65-.19l.22.72a6.34,6.34,0,0,1-.88.25,4.58,4.58,0,0,1-1,.1,2.88,2.88,0,0,1-1.07-.17,1.67,1.67,0,0,1-.69-.49,2,2,0,0,1-.37-.76,4,4,0,0,1-.11-1Z" fill="#54a3d9"/><path d="M94.71,291.58a4.49,4.49,0,0,0,1.06.58,3.79,3.79,0,0,0,1.36.22,2.1,2.1,0,0,0,1.18-.26.77.77,0,0,0,.38-.65.73.73,0,0,0-.41-.6,5.47,5.47,0,0,0-1.48-.53,6.35,6.35,0,0,1-.86-.26,3.27,3.27,0,0,1-.7-.37,2,2,0,0,1-.48-.46.94.94,0,0,1-.18-.56,1.43,1.43,0,0,1,.16-.65,1.49,1.49,0,0,1,.46-.53,2.36,2.36,0,0,1,.77-.35A4,4,0,0,1,97,287a3.82,3.82,0,0,1,1.28.22,4.47,4.47,0,0,1,1.06.52l-.48.64A3.46,3.46,0,0,0,98,288a3.27,3.27,0,0,0-1.69-.1,1.35,1.35,0,0,0-.44.18.67.67,0,0,0-.24.27.76.76,0,0,0-.07.32q0,.36.49.59a7.31,7.31,0,0,0,1.28.44,5.82,5.82,0,0,1,1.76.68,1.21,1.21,0,0,1,.57,1.06,1.39,1.39,0,0,1-.17.67,1.66,1.66,0,0,1-.5.55,2.56,2.56,0,0,1-.82.38,4.08,4.08,0,0,1-1.11.14,4.51,4.51,0,0,1-1.59-.27,5.75,5.75,0,0,1-1.27-.64Z" fill="#54a3d9"/><path d="M103.19,287.83a1,1,0,0,1,.28-.74,1,1,0,0,1,1.38,0,1,1,0,0,1,.28.74,1,1,0,0,1-.28.73,1,1,0,0,1-1.38,0A1,1,0,0,1,103.19,287.83Zm0,4.3a1,1,0,0,1,.28-.74,1,1,0,0,1,1.38,0,1,1,0,0,1,.28.74,1,1,0,0,1-.28.73,1,1,0,0,1-1.38,0A1,1,0,0,1,103.19,292.13Z" fill="#54a3d9"/><path d="M80,303.06h5.16v.74H80Z" fill="#54a3d9"/><path d="M94.48,301.56h.82l.08,1h0a4.68,4.68,0,0,1,1-.82,2.27,2.27,0,0,1,1.21-.32,1.86,1.86,0,0,1,1.53.59,2.73,2.73,0,0,1,.49,1.76v3.62h-1V303.9a2,2,0,0,0-.31-1.23,1.23,1.23,0,0,0-1-.4,1.7,1.7,0,0,0-.49.07,1.86,1.86,0,0,0-.44.2,3.33,3.33,0,0,0-.44.33q-.23.2-.48.46v4.07h-1Z" fill="#54a3d9"/><path d="M101.54,305.88a1.56,1.56,0,0,1,.23-.86,1.86,1.86,0,0,1,.74-.63,5.22,5.22,0,0,1,1.31-.43,16.66,16.66,0,0,1,1.93-.26,2.13,2.13,0,0,0-.1-.56,1.23,1.23,0,0,0-.26-.46,1.29,1.29,0,0,0-.47-.32,1.88,1.88,0,0,0-.71-.12,2.83,2.83,0,0,0-1.1.22,6.85,6.85,0,0,0-.94.47l-.38-.67a5.45,5.45,0,0,1,.51-.29,5.66,5.66,0,0,1,.62-.27,5.26,5.26,0,0,1,.7-.2,3.69,3.69,0,0,1,.76-.08,2.27,2.27,0,0,1,1.78.65,2.53,2.53,0,0,1,.58,1.75v3.58h-.8l-.08-.79h0a5.63,5.63,0,0,1-1.09.66,2.87,2.87,0,0,1-1
```

Comment:
Ah, I should have called this out. We are going to use a different config and fix the capitalization across the board.

The manual vs automatic distinction is trying to show that manual "pipelines" are the most painful way to use CLCs. Tools like Matchbox (and eventually Terraform) will transparently transpile the config for the end user.